### ISSUE TITLE
[dev]: add aur job to publish.yaml

### ISSUE THREAD

**Issue by jsjoeio** [coder/code-server#2845]
## Problem

After we publish a release, we have to manually go to the [code-server-aur](https://github.com/cdr/code-server-aur) repo and publish a new version of the package. This should be automated as part of the release process.

## Solution

Add a job called `aur` to `publish.yaml` that waits on `npm` and then publish new AUR package with script.

### Notes
- this may not be straightforward. we'll need to add a CI account or something to the AUR package as a maintainer
- it could make sense for the job here to open a PR then have a separate GH Actions workflow in the actual repo that publishes the package on merge into default branch
- this could help: https://github.com/ouuan/AUR-template

### Related

- https://github.com/cdr/code-server/issues/2802

**Comment by jsjoeio** [comment]
Still planning to do this

**Comment by jsjoeio** [comment]
I am hoping to work on this soon.

**Comment by jsjoeio** [comment]
Closing in favor of https://github.com/coder/code-server/issues/5366 and https://github.com/coder/code-server-aur/issues/13
